### DISCUSSION TITLE
About this new Rust Trademark Policy Stuff.

### DISCUSSION THREAD

**Discussion by Kambaa** [rustdesk/rustdesk#4022] [Q&A]
Hello, is this project gonna be effected from the new Rust Trademark stuff? Can somebody please check, explain and ease my concerns?

Something about this policy is restricting even the use of 'Rust'  on domain names? What the heck?

I love this project, and I don't want this to impact any discouragement to people who are working on this awesome project. Love you all! 

Have a nice day.

https://twitter.com/rust_foundation/status/1644132378858729474

https://youtu.be/gutR_LNoZw0?t=162

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1646358323644465152

**Comment by Kambaa** [comment]
Update: https://blog.rust-lang.org/inside-rust/2023/04/12/trademark-policy-draft-feedback.html
It seems the with the next update on the policy, our questions will have answers. Better wait patiently.
Have a good one!
